### PR TITLE
PLATEXP1729: Modified add-disk-space-to-server-or-hypervisor

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/PLATEXP-1729

@ssalinas24 @Rohi2305 

Please review the article.

### CODE REVIEW

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 1]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
...
 
-   {{<image src="adddisk1.png" alt="" title="">}}
+<img width="677" alt="Select Add Disk Space to Server or Hypervisor from common request template" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk6.png">
 
-6. In the **Ticket Details** section, select a device from the drop-down menu.
+**Step 4.** In **Issue Details** section, select a device from the drop-down menu.
 
-7. Enter the amount of disk space needed, and choose whether to add the space to an existing volume or to
-   configure as a new drive.
+<img width="440" alt="Select required device from drop-down menu" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk8.png">
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:33]: This image shows real device number, and names.
Did you alter the information before taking the screenshot?

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 2]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
...
+<img width="941" alt="Select support center from top navigation bar" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk5.png">
 
-5. From the drop-down menu, select **Add Disk Space to Server or Hypervisor.**
+**Step 3.** From the **Common Request** menu select **Add Disk Space to Sever or Hypervisor** as shown in following image.
 
-   {{<image src="adddisk1.png" alt="" title="">}}
+<img width="677" alt="Select Add Disk Space to Server or Hypervisor from common request template" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk6.png">
 
-6. In the **Ticket Details** section, select a device from the drop-down menu.
+**Step 4.** In **Issue Details** section, select a device from the drop-down menu.
```
Comment (ssalinas24) [inline content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:31]: `In the **Issue Details** section`.. add `the`

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md (hunk 1 above)
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:33]: Replace image as suggested.

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md (hunk 2 above)
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:31]: added.

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 3]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
 
 ### Create a ticket in the MyRackspace portal
 
-1. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with your username and
+**Step 1.** Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with your username and
    password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+<img width="409" alt="Enter credentials for Username and Password" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk4.png">
```
Comment (Rohi2305) [inline content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:21]: All images are rendering.

**Review by Rohi2305 — COMMENTED** [review]
Re-arrange all images and do the required changes in steps.

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 4]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
...
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** In the top navigation bar, choose **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+<img width="941" alt="Select support center from top navigation bar" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk5.png">
 
-5. From the drop-down menu, select **Add Disk Space to Server or Hypervisor.**
+**Step 3.** From the **Common Request** menu select **Add Disk Space to Sever or Hypervisor** as shown in following image.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:27]: Done

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 5]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
...
 
-   {{<image src="adddisk1.png" alt="" title="">}}
+<img width="677" alt="Select Add Disk Space to Server or Hypervisor from common request template" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk6.png">
 
-6. In the **Ticket Details** section, select a device from the drop-down menu.
+**Step 4.** In the **Issue Details** section, select a device from the drop-down menu.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:31]: Merged step 4, 5, 6, & 7

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md [hunk 6]
```diff
@@ -15,31 +15,40 @@ This article explains how to create a ticket to add disk space to a server or hy
...
 
-7. Enter the amount of disk space needed, and choose whether to add the space to an existing volume or to
-   configure as a new drive.
+<img width="447" alt="Select required device from drop-down menu" src="/support/how-to/add-disk-space-to-server-or-hypervisor/adddisk8.png">
 
-   {{<image src="adddisk2.png" alt="" title="">}}
+**Step 6.** In **Add to Existing or New Volume** choose whether to add the space to an existing volume or to configure as a new drive.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:35]: Modified as "In **Add to Existing or New Volume** select the radio button to add the space to an existing volume or to configure as a new drive."

File: content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md (hunk 3 above)
Comment (asmita32675) [reply content/how-to/dedicated-hosting/add-disk-space-to-server-or-hypervisor/index.md:20]: Done

**Review by Rohi2305 — APPROVED** [review]

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**